Personal Rationale (#29)
graceuche/northeastern-final-project: README.md

### Personal Rationale

> My name is Grace Uche. I am a Professional Registered Health Information Administrator with a Master Degree in Business Administration. I’m also a Licensed Health and Life Insurance Broker. As a Health Information Management professional, for over 15 years, I have had the opportunity to practice and work in different healthcare settings. Most of my work has been in educating and training  students in both Associates and Bachelors degrees in the Health Information Management programs to successfully complete the program, sit and pass any of the national certificate exams, and be able to obtain a meaningful job in the field.

Reviewer: ```suggestion
My name is Grace Uche. I am a Professional Registered Health Information Administrator with a Master Degree in Business Administration. I am also a Licensed Health and Life Insurance Broker. As a Health Information Management professional, for over 15 years, I have had the opportunity to practice and work in different healthcare settings. Most of my work has been in educating and training  students in both Associates and Bachelors degrees in the Health Information Management programs. I help students to successfully complete the program, sit and pass the national certification exams, and obtain a job in the field.
```

> The  quest for knowledge drove me into Health and Life Insurance because I wanted to understand how insurance company carriers operate, factors that determine insured insurability and coverage suitability, as well as to understand those situations that might deter individuals from not being able to get coverage.

Reviewer: ```suggestion
The  quest for knowledge drove me into Health and Life Insurance because I wanted to understand how insurance companies operate, factors that determine insurability and coverage suitability. Furthermore, I wanted to explore situations that might deter individuals from getting coverage.
```

> As an educator, I have successfully taught courses in Data Management which encompases data collection, storage, retrieval, aggregation, and interpretation, but not as far as using machine learning and models to write codes and to predict desirable outcomes.

Reviewer: ```suggestion
As an educator, I have successfully taught courses in Data Management. However, as a coder I was an absolute beginner 12 weeks ago. Therefore, I was set to produce a final project with an extensive Exploratory Data Analysis section which should allow me to practice and to demonstrate  my newly acquired Python skills.
```

> The fact that healthcare is data driven led me towards Data Analytics, and the idea is to help add more value to my existing knowledge in both healthcare and business. I also must mention that a Data Analytics certificate would enable me to gain meaningful employment with a company that would utilize my wealth of experiences and knowledge to grow their company.  

Reviewer: ```suggestion
During the 12 weeks of the Data Analytics Career Accelerator course I developed passion for Data Visualization, so another on of my personal goals was to explore various Data Visualization libraries in Python and assess their suitability for various stages of Exploratory Data Analysis.
```

> As always, time flies when you are having fun! I cannot believe that I am working on my final project and presentation in less than a week. It feels like the course just started yesterday! But, it is all good because I have learned so much about datasets, data exploration, visualization (my favorite part), and predictions. 

Reviewer: ```suggestion
To sum up, I set to myself the following personal goals:
- gain personal knowledge about the health insurability of the American population by exploring a representative sample of the USA population
- visualize a representative sample of the USA population by factors that determine health insurance charges
- explore summative statistics of the representative sample of the USA population by factors that determine health insurance charges
- compare some machine learning algorithms that allow predicting health insurance charges
```

> Regardless of which company or field I ended up working with today, one thing is certain, and that is, I would use my wealth of knowledge and experiences to the best of my ability to add value to the company.

Reviewer: ```suggestion
I wanted to gain more knowledge and experiences in the health insurance sector that could transfer into various roles..
```
